cli/item_to_function.py

```diff
@@ -188,6 +200,8 @@ def create_function_yaml(
 
     function_object.export(target=str(output_path.resolve()))
 
+    remove_build_info_from_yaml(function_path=output_path)
```

Comment:
Why do this *after* you export the yaml file? Better to just remove the values from `function_object` before exporting it...

Reply:
i agree in order to do that we need to change 
https://github.com/mlrun/mlrun/blob/development/mlrun/runtimes/base.py
this a workaround

Reply:
Why change the export? You can just go and do `function_object.spec.build.code_origin = ''` (and the same for `origin_filename`). Doing this here, rather than inside the `export()` actually makes sure you don't touch any of the regular MLRun flows, but rather just override the behavior for the hub purposes.

Reply:
ok fixed in the new PR